Makefile

```diff
@@ -224,24 +263,19 @@ package-windows: require-version windows-386
 	docker run -v $$PWD:/flashlight-build -v $$SECRETS_DIR:/secrets -t $(DOCKER_IMAGE_TAG) /bin/bash -c 'cd /flashlight-build && BNS_CERT="/secrets/bns_cert.p12" BNS_CERT_PASS="'$$BNS_CERT_PASS'" VERSION="'$$VERSION'" make docker-package-windows' && \
 	echo "-> lantern-installer.exe"
 
-package-darwin: require-version darwin
+package-darwin: require-version require-npm require-appdmg require-svgexport darwin
```

Comment:
Oh sorry -- was looking at the deleted code =). Yeah looks like maybe we no longer directly need npm.